src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -25,42 +27,55 @@ def get(self, request, organization):
         except NoProjects:
             return Response([])
 
-        aggregate_column = request.GET.get("aggregateColumn", "duration")
-        orderby = request.GET.get("order", None)
+        filter_query = request.GET.get("query")
+        aggregate_column = request.GET.get("aggregateColumn")
+
+        ALLOWED_AGGREGATE_COLUMNS = {
+            "transaction.duration",
+            "measurements.lcp",
+            "spans.browser",
+            "spans.http",
+            "spans.db",
+            "spans.resource",
+        }
```

Comment:
nit: Can we move constants out of this class function (like after the imports basically)